redash/models.py

```diff
@@ -613,6 +613,7 @@ class Alert(ModelTimestampsMixin, BaseModel):
     options = JSONField()
     state = peewee.CharField(default=UNKNOWN_STATE)
     last_triggered_at = DateTimeTZField(null=True)
+    rearm = peewee.IntegerField(null=True)
```

Comment:
Oops, ignore that - it's not a boolean (maybe I shouldn't do code reviews at 11PM...).